### DISCUSSION TITLE
2.3.0

### DISCUSSION THREAD

**Discussion by T8RIN** [T8RIN/ImageToolbox#410] [Announcements]
## What's Changed
* Added info for presets unit by #348
* Random filename option added by #352 and #355 
* Moved telegram preset to "Presets" block card
* Added ability to use shape masks when cropping
    - Rounded Corners
    - Cut Corners
    - Oval
    - Squircle
    - Octagon
    - Rounded Pentagon
    - Clover
    - David Star
    - Kotlin Logo
    - Heart
    - Star
    - Image Mask
* Added backup and restore option in settings
* Updated color picker sheet
* Added in app font scale changing option
* Now you can select inApp font from list below
  - Montserrat
  - Comfortaa
  - Caveat
  - Handjet
  - Jura
  - Podkova
  - Tektur
  - YsabeauSC
* Imroved animations
* Emoji showing changed and also new ones added
* Added lower api version support by #387 
* Added Background remover by #293 with draw and auto processing options
* Imroved Draw Screen, added new drawing types including Neon and Highlighter
* Added blur aka spread option for drawing
* Added transparency checker for images
* Compare option now can share and save splitted image
* New Crash Screen with report issue ability
* now BatchResizeScreen is named ResizeAndConvert it replaced single resize and batch resize for more intuitive UX
* Single Edit mode like any gallery has - EXIF/Cropping/Filtering/Drawing/Background Eraser/Resizing/Converting in one screen
* New intuitive Color Chooser instead if RGB sliders
* Added approximate color names for colors in Palette Screen, also it will be displayed in Color Picker Pipette screen
* Added Firebase Analytics and Crashlytics which also can be disabled

## What's Fixed
* Fix filename saving by #353 
* Fix presets weird behavior in #359 and #358 
* Fixed presets displaying
* Fix UI by #389
* Fixed BytesrResize option by #382 
* Fixed #403
* Fixed abnormal screen rotation when selecting font size by #405
* Code refactors and other bug fixes

## New Contributors
* @V00VA made their first contribution in https://github.com/T8RIN/ImageToolbox/pull/349

**Full Changelog**: https://github.com/T8RIN/ImageToolbox/compare/2.2.1...2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/T8RIN/ImageToolbox/releases/tag/2.3.0'>2.3.0</a>.</em>

**Comment by xventivs** [comment]
❤️‍🔥😃😊
